go/pvl/interp.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 // UsePvl says whether to use PVL for verifying proofs.
-const UsePvl = false
+const UsePvl = true
```

Comment:
Affects [this](https://github.com/keybase/client/blob/master/go/externals/proof_support_coinbase.go#L36) and [that](https://github.com/keybase/client/blob/master/go/externals/proof_support_coinbase.go#L52) in all external checkers.